mu-plugins/blocks/time/src/index.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * WordPress dependencies
+ */
+import { registerFormatType, toggleFormat } from '@wordpress/rich-text';
+import { RichTextToolbarButton } from '@wordpress/block-editor';
+
+/**
+ * Internal dependencies
+ */
+import metadata from './block.json';
+
+const Edit = ( { isActive, onChange, value } ) => (
+	<RichTextToolbarButton
+		icon={ metadata.icon }
+		title={ metadata.title }
+		onClick={ () => {
+			onChange(
+				toggleFormat( value, {
+					type: metadata.name,
+				} )
+			);
+		} }
+		isActive={ isActive }
+	/>
+);
+
+registerFormatType( metadata.name, {
+	title: metadata.title,
+	tagName: 'a',
+	className: 'wporg-time',
```

Comment:
This creates a link that doesn't do anything — could you maybe use the `time` element instead? Use the PHP to add the `datetime` attribute, and then use that to build up the element in JS?

Reply:
Done